tianshou/utils/optim.py

```diff
@@ -20,3 +23,42 @@ def optim_step(
     if max_grad_norm:
```

Comment:
this `optim_step` is not covered by any test

Reply:
Yes, it slipped in with a previous merge, where I have forgotten to delete it. I use it on other branches in my fork, and plan to include tests for it and use it across policies (this code is duplicated many times over). Can we keep it for the time being? It is currently unused

Reply:
sure